Tools/_OpenSpace/update_tts_voices.py

```
@@ -0,0 +1,106 @@
+def loc_key(speaker: str) -> str:
+    return f"tts-voice-name-{speaker}"
+
+
+def build_yaml(voices: list[dict]) -> str:
+    lines = []
+    for voice in voices:
+        speaker = voice["speakers"][0]
+        gender  = voice.get("gender", "none")
+        key     = loc_key(speaker)
+
+        lines.append(f"- type: ttsVoice")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)